models/user.js

```diff
@@ -8,43 +9,61 @@ const bcrypt = require('bcryptjs');
 const uniqueValidator = require('mongoose-unique-validator');
 
 const UserSchema = mongoose.Schema({
-  createdAt              : { type: Date  },
-  updatedAt              : { type: Date  },
+  createdAt              : { type: Date, default: Date.now },
```

Comment:
new Date() vs Date.now: functionally similar except Date.now is not supported by Internet Explorer 8.

Reply:
that's good to know!